### PR TITLE
NCD TTB CMS

### PR DESCRIPTION
**Pull request by t7phy**

### CODE REVIEW

**Comment by t7phy** [comment]
@scarlehoff the newly implemented ones are:  CMS 13 TEV LJ (all in the metadata) and CMS_TTBAR_8TEV_LJ_DIF_PTT-NORM

File: buildmaster/CMS_TTBAR_13TEV_LJ_DIF/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+      kinematics_override: identity
+      x_scale: log
+      plot_x: pT_t
+      figure_by: 
+        - sqrt_s
+    theory:
+      FK_tables:
+        - - CMS_TTBAR_13TEV_LJ_DIF_PTT
+        - - CMS_TTBAR_13TEV_TOT_X-SEC
+      opeartion: 'ratio'
```
Comment (scarlehoff) [inline buildmaster/CMS_TTBAR_13TEV_LJ_DIF/metadata.yaml:244]:
```suggestion
      operation: 'ratio'
```

File: buildmaster/CMS_TTBAR_8TEV_LJ_DIF/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        mt_sqr: {description: "mass of top quark squared", label: '$m_{t}^2$', units: '$GeV^2$'}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics_dSig_dpTt_norm.yaml
+    data_central: data_dSig_dpTt_norm.yaml
+    data_uncertainties:
+      - uncertainties_dSig_dpTt_norm.yaml
+    kinematic_coverage: [pT_t, mt_sqr, sqrt_s]
+    plotting:
+      dataset_label: 'CMS 8 TeV $t\bar{t}$: $\frac{1}{\sigma}\frac{d\sigma}{dpT_t}$'
+      plot_x: pT
```
Comment (scarlehoff) [inline buildmaster/CMS_TTBAR_8TEV_LJ_DIF/metadata.yaml:37]:
```suggestion
      plot_x: pT_t
      kinematics_override: identity
```

**Comment by scarlehoff** [comment]
Report for the new datasets https://vp.nnpdf.science/iooLV2_jQQOMQrX3LnnJTA==

**Comment by scarlehoff** [comment]
Of the data-problematic ones, only CMS_TTBAR_2D_DIFF_MTT_TRAP_NORM is used in NNPDF4.0

Coincidentally, this is the one for which the new pipeline produces different results so not sure whether something needs to be done.

**Pull request closed without merging**